src/VSCodeExtension/src/generator-qsharp/app/index.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+'use strict';
+const Generator = require('yeoman-generator');
+const chalk = require('chalk');
+const yosay = require('yosay');
+
+module.exports = class extends Generator {
```

Comment:
Similarly here, I'd recommend using TypeScript's `export` modifier keyword to directly setting `module.exports`. You can  also use the `default` modifier to make it the only exported definition.

I think this is right, but you'll want to double-check.
```suggestion
export default class extends Generator {
```